python/run-tests.py

```diff
@@ -182,4 +188,7 @@ def run_delta_connect_codegen_python(root_dir):
     run_mypy_tests(root_dir)
     run_pypi_packaging_tests(root_dir)
     run_delta_connect_codegen_python(root_dir)
-    test(root_dir, package)
+    test(root_dir, "delta", package)
+    is_spark_master = os.getenv("IS_SPARK_MASTER")
+    if is_spark_master is not None:
+        test(root_dir, path.join("delta", "connect"), package)
```

Comment:
What's the significance of this path?

Reply:
Before, we only run tests in `python/delta/tests`, 

`test_files = [os.path.join(test_dir, f) for f in os.listdir(test_dir)
                  if os.path.isfile(os.path.join(test_dir, f)) and
                  f.endswith(".py") and not f.startswith("_")]` does not recursively go through the whole `delta` folder while we are adding tests to `python/delta/connect/tests`, so